README.md

```diff
@@ -63,6 +63,17 @@ cmake --build . --config Release
 
 If everything went OK, `bin\Release\rwkv.dll` file should appear.
 
+##### WIndows + CuBlas
+copy cuda libs into release
+https://github.com/ggerganov/llama.cpp/pull/1271/files
```

Comment:
It can be unclear for users where the actual files are. I suggest leaving a link to a specific release that contains CUDA dlls: https://github.com/ggerganov/llama.cpp/releases/tag/master-fab49c6